### ISSUE TITLE
flow charts: email shape does not render multiple lines

### ISSUE THREAD

**Issue by tsiq-karold** [testingisdocumenting/znai#645]
Example flow chart json:
```json
{
  "nodes": [
    {
      "id": "foo",
      "label": "foo\nbar"
    },
    {
      "id": "bar",
      "label": "foo\nbar",
      "shape": "email"
    }
  ],
  "edges": [
    ["foo", "bar"]
  ]
}
```
Rendered flow chart:
![Screenshot 2021-06-28 at 11 22 30](https://user-images.githubusercontent.com/33879397/123621487-29a34700-d803-11eb-8ed2-6c9fd0742d26.png)

As you can see the default shape renders the multiple lines correctly, the email shape does not.

**Comment by tsiq-karold** [comment]
I've also encountered this with the `database` shape so I wonder if this is a problem with anything other than the default rectangle.

**Comment by suhasdara** [comment]
I have observed the same issue when specifying `colorGroup`

**Comment by MykolaGolubyev** [comment]
@suhasdara thanks for letting me know. Do you use a regular shape? Any chance you can elaborate on your full use case?

**Comment by suhasdara** [comment]
Yes I use a regular shape but with different colors. Essentially I need my nodes to be one of two colors. My labels also contain the newline character and I have exactly the same issue as what the original person reported when explicitly setting colorGroup: "b". I tried using highlight instead of colorGroup and that causes the same problem as well. Essentially, any formatting of the node other than the default seems to cause the problem.

**Comment by MykolaGolubyev** [comment]
@suhasdara thank you for the pointers. I am able to reproduce now. Working on a fix.
Still curios what do you use diagrams for! 
If you have a moment, could you please post here - https://github.com/testingisdocumenting/znai/discussions/824 ?

**Comment by MykolaGolubyev** [comment]
this has been released as 1.62: https://testingisdocumenting.org/znai/release-notes/2022#1-62
